src/main/java/graphql/analysis/QueryTraverser.java

```diff
@@ -51,32 +51,38 @@ public class QueryTraverser {
     private QueryTraverser(GraphQLSchema schema,
                            Document document,
                            String operation,
-                           Map<String, Object> variables) {
-        assertNotNull(document, () -> "document can't be null");
+                           CoercedVariables coercedVariables) {
+        this.schema = schema;
```

Comment:
did you get rid of `assertNotNull(document, () -> "document can't be null");` accidentilly here??

Reply:
You probably saw it later - the null assert is inside the builder now